change/@fluentui-react-slider-3db1e642-9ac3-4cc6-b2c1-8d6f1cce0f12.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Cleaning up the Slider utils folder.",
+  "packageName": "@fluentui/react-slider",
+  "email": "czearing@outlook.com",
+  "dependentChangeType": "none"
```

Since this is actual code not just tests
```suggestion
  "type": "prerelease",
  "comment": "Cleaning up the Slider utils folder.",
  "packageName": "@fluentui/react-slider",
  "email": "czearing@outlook.com",
  "dependentChangeType": "patch"
```